### ISSUE TITLE
Dota 2 crashes at loading screen, after crash won't let me open other games

### ISSUE THREAD

**Issue by LexMK** [ValveSoftware/Dota-2#1662]
When I try to open the game, it crashes during the initial loading screen. After the crash, Steam won't open other games (It shows the "Prepairing to launch" windows, but stops there) unless I reboot the computer. Restarting Steam won't fix. Only after rebooting I can open other games. No other game behaves like that. Everything else works. This started happening right after the last update. Before the update, Dota worked as expected. I searched on the net and it seems I'm not the only one with this problem. It seems to be a Linux-only issue, but can't be sure about that.

I use Linux Mint Debian Edition, all packages updated. I use AMD proprietary driver on a HD 7750 GPU.

**Comment by MaximilianoFelice** [comment]
Still having this problem. According to console log, the game seems to be trying to load DirectX files in Linux based systems, which obviously leads to a dead end while instantiating a D3DX device, causing the game to crash.

    [...]
    Creating D3D9 device with D3DCREATE_MULTITHREADED
    IDirect3DDevice9::Create: BackBufWidth: 1920, BackBufHeight: 1080, D3DFMT: 3, BackBufCount: 1. MultisampleType: 0, MultisampleQuality: 0
    [...]

**Comment by Tele42** [comment]
@MaximilianoFelice This is expected as Source games still have a d3d9 engine, which is feed into ToGL which translates it to opengl 2.1.
